### PR TITLE
Add DB api and sqlalchemy support

### LINKED ISSUE #68: DB API and SQLAlchemy support
And when working with other community tools such as LangChain, Querybook, Superset, SQLAlchemy plays as the driver role.
It is better that Timeplus python sdk can support it.

- [pep-0249](https://peps.python.org/pep-0249/)
- https://medium.com/@er.26yashiagarwal/python-database-api-beb8e61109cf
-  https://www.sqlalchemy.org/

### PR DESCRIPTION
**Pull request by gangtao**
Add sql analyze api to query
Add view api

Add DB API basic support (query only)
Add sqlalchemy timeplus dialect, tested `1.4.x` and `2.0.6`

### CODE REVIEW

File: python/timeplus/api.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
+import json
```
Comment (ye11ow) [inline python/timeplus/api.py]: `api` is too generic, shall we call it `dbAPI` or something else? Maybe I'm not very familiar with this, I think `dbAPI` is too generic as well.

Comment (ye11ow) [reply python/timeplus/api.py]: Also, shall we put it into `superpartner` repo? IMO this is a very PoC code, we probably should not put it into SDK (which is supposed to be stable)

Comment (gangtao) [reply python/timeplus/api.py]: > Also, shall we put it into `superpartner` repo? IMO this is a very PoC code, we probably should not put it into SDK (which is supposed to be stable)

I found some other db libs put them together, so it should be fine here

**Review by jovezhong — APPROVED** [review]

**Comment by jovezhong** [comment]
(Jove Github Bot) added it to the current sprint.

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**